maestrano-connector-rails.gemspec

```diff
@@ -31,7 +31,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('bootstrap-sass')
   s.add_runtime_dependency('config')
   # v1.1 breaks the build. See: https://github.com/ruby-concurrency/concurrent-ruby/issues/768
-  s.add_runtime_dependency('concurrent-ruby', '~> 1.0.5')
+  s.add_runtime_dependency('concurrent-ruby', '1.0.5')
```

Comment:
`~> 1.0.5` shouldn't have let you upgrade to 1.1?
Is `1.1.1` fixing it?

Reply:
~> 1.0.5 upgraded to 1.1.3 . 1.1.1 does not solve it so I am locking it to 1.0.5 for now.

Reply:
As discussed this shouldn't be needed. See if it's better with `v2.3.6`